actions/ansible-docs-build-init/action.yml

```diff
@@ -74,7 +80,7 @@ runs:
       run: |
         echo "::group::Installing antsibull-docs"
         if [[ "${{ inputs.antsibull-docs-version }}" != "" ]] ; then
-            pip install https://github.com/ansible-community/antsibull-docs/archive/${{ inputs.antsibull-docs-version }}.tar.gz
+            pip install https://github.com/${{ inputs.antsibull-docs-repository || 'ansible-community/antsibull-docs' }}/archive/${{ inputs.antsibull-docs-version }}.tar.gz
```

Comment:
Is the `||` needed since the input has a default value?

Reply:
Probably not, but I'm not that familiar with the GH configuration system, and I don't think there is an analogue to the `| omit` feature from Ansible, so calling workflows (or composite actions) might pass in `''` if they don't want to pass a value, in which case this `||` would make sure the proper default is use.

I can remove this though if you think this shouldn't be necessary.

Reply:
I was going to say 

> I think if we specify a default, then we should not handle an explicitly passed in `''` unless it has its own specific behavior.

But I think you have a point about calling workflows/composite actions 😕
Perhaps those _should have to_ duplicate the default in themselves if they want to handle that... as it would give them a stable interface even if the underlying default changes, but I can definitely see how pass-thru is desired too...

I don't recall how we handle this in other parameters, maybe we should look to be consistent.

In the end I will trust your judgement on it. If we keep the current behavior, please add a note in the description that calls out the effect of explicitly passing an empty string. Thanks!

Reply:
I decided to remove this, since something similar wasn't done anywhere else in these actions.